cmd/node/node.go

```diff
@@ -326,11 +326,11 @@ func (app *SpacemeshApp) getAppInfo() string {
 }
 
 // Cleanup stops all app services
-func (app *SpacemeshApp) Cleanup(cmd *cobra.Command, args []string) {
-	log.Info("App Cleanup starting...")
+func (app *SpacemeshApp) Cleanup(_ *cobra.Command, _ []string) {
```

Comment:
Removing the names entirely is even cleaner IMO.

```suggestion
func (app *SpacemeshApp) Cleanup(*cobra.Command, []string) {
```